usaspending_api/recipient/models.py

```diff
@@ -170,10 +170,9 @@ class Meta:
 
 
 class RecipientAgency(models.Model):
-    id = models.BigAutoField(primary_key=True)
     fiscal_year = models.IntegerField(db_index=True)
     toptier_code = models.TextField(db_index=True)
-    recipient_hash = models.UUIDField(db_index=True)
+    recipient_hash = models.UUIDField(primary_key=True, db_index=True)
```

Comment:
Aren't there duplicates on `recipient_hash`? If we don't really care about the `id` field then you can omit that field, don't add `primary_key` to anything and it will automatically create a PK.